### PR TITLE
After adding frontend for logging errors

### PR DESCRIPTION
**Pull request by dhruv1794**

### CODE REVIEW

**Comment by aholdahl** [comment]
@hardikk08 would you also take a look at this MR since you originally reported the task?

File: src/classBased/sezzle-init.js [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+try  {
+    new SezzleJS(document.sezzleConfig).init();
+} catch(e) {
+    logErrors(e.message)
+}
+
+function logErrors(message) {
+    const trackingURL = document.widgetServerBaseUrl ? `${document.widgetServerBaseUrl}/v1/event/log` : 'https://widget.sezzle.com/v1/event/log';
+    const body = {
+        error: message
```
Comment (hardikk08) [inline src/classBased/sezzle-init.js:13]: we will need to specify an event type too right? @dhruv1794

Comment (dhruv1794) [reply src/classBased/sezzle-init.js]: made this change in my latest  commit.

File: src/classBased/sezzle-init.js [hunk 2]
```diff
@@ -1,4 +1,36 @@
 import SezzleJS from './sezzle';
 
 // Assumes document.sezzleConfig is present
-new SezzleJS(document.sezzleConfig).init();
+try  {
+    new SezzleJS(document.sezzleConfig).init();
+} catch(e) {
+    logErrors(e.message)
+}
+console.log(document.sezz)
```
Comment (aholdahl) [inline src/classBased/sezzle-init.js:9]: Can we remove this console.log?

Comment (dhruv1794) [reply src/classBased/sezzle-init.js]: its done

**Review by aholdahl — APPROVED** [review]
@hardikk08 interested in your additional review

**Review by hardikk08 — APPROVED** [review]

**Pull request merged**